packages/blockchain-tools/src/contracts/solidity/erc721/erc721inc.sol

```diff
@@ -17,7 +17,7 @@ contract NFToken is ERC721, ERC721Enumerable, ERC721URIStorage, ERC721Burnable,
     {}
 
     function _baseURI() internal pure override returns (string memory) {
-        return "https://cfce.io";
+        return "";
```

Comment:
What is the purpose of this, and the purpose of the change?

Reply:
The URI can be both a url like https://cfce.io/nft/123 or ipfs:123 and since we use ipfs, then the url prefix is not needed. The method returns the ipfs without prefix. It is good practice to use permanent storage like ipfs, but some minters just use their website domain for that, thats why both options are valid, but permanent storage preferred